assets/app/scripts/controllers/overview.js

```diff
@@ -58,6 +60,26 @@ angular.module('openshiftConsole')
     $scope.renderOptions = $scope.renderOptions || {};
     $scope.renderOptions.showSidebarRight = false;
 
+    /*
+     * HACK: The use of <base href="/"> that is encouraged by angular is
+     * a cop-out. It breaks a number of real world use cases, including
+     * local xlink:href. Use location.href to get around it, even though
+     * these SVG <defs> are local in the template.
```

Comment:
@liggitt Thoughts on this?

@stefwalter Forgive my ignorance, but I'm assuming relative URIs like `"#vertex-DeploymentConfig"` won't work here?